src/Components/Onboarding/Steps/Artists/index.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import SelectableItemContainer from "./SelectableItemContainer"
+
+import { StepProps } from "../../Types"
+import { Layout } from "../Layout"
+
+export interface RelayProps {
+  popular_artists: {
+    artists: any[]
+  }
+}
+
+/**
+ * Artists
+ * -> Input
+ *    List
+ *    -> Popular
+ *       Search
+ */
+
```

Comment:
This can be removed, it’s already in your excellent PR write-up, feel free to merge either way, up to you 👍